packages/test/patches/0005-gh-93839-Move-Lib-ctypes-test-to-Lib-test-test_ctype.patch

```diff
@@ -0,0 +1,753 @@
+From d82e0bfe8b98a122ca443b356d81998c804b686e Mon Sep 17 00:00:00 2001
+From: Victor Stinner <vstinner@python.org>
+Date: Tue, 21 Jun 2022 10:24:33 +0200
+Subject: [PATCH 5/9] gh-93839: Move Lib/ctypes/test/ to Lib/test/test_ctypes/
+ (#94041)
+
+* Move Lib/ctypes/test/ to Lib/test/test_ctypes/
+* Remove Lib/test/test_ctypes.py
```

Comment:
Could you explain why the above two are necessary? Can't we keep it in that folder?

Reply:
These patches are necessary to run ctypes, unittest tests. We apply these patches when building CPython, so it also needs to be applied here (https://github.com/pyodide/pyodide/pull/3507). This is a bit annoying, but we will be able to remove these patches when we update to Python 3.12.